crates/database/src/onchain_broadcasted_orders.rs

```diff
@@ -7,27 +7,21 @@ use {
 #[derive(Clone, Debug, Eq, PartialEq, sqlx::Type, strum::EnumIter)]
 #[sqlx(type_name = "OnchainOrderPlacementError", rename_all = "snake_case")]
 pub enum OnchainOrderPlacementError {
-    QuoteNotFound,
-    InvalidQuote,
```

Comment:
I think removing these variants here will cause the `OPEN_ORDERS` query to fail when it tries to fetch the `onchain_placement_error` column for orders which already have these removed error variants stored.